components/raftstore/src/store/worker/pd.rs

```diff
@@ -427,6 +437,7 @@ where
 
 const HOTSPOT_KEY_RATE_THRESHOLD: u64 = 128;
 const HOTSPOT_BYTE_RATE_THRESHOLD: u64 = 8 * 1024;
+const HOTSPOT_QUERY_RATE_THRESHOLD: u64 = 8; // TODO: need more test
```

Comment:
I think it's too small, assume the bytes 1 key have 64 bytes,  it should be 128.